e2e/tests/helpers.ts

```diff
@@ -46,7 +46,7 @@ export const login = async (page: Page) => {
     const newPage = await context.newPage();
 
     // navigate to MailHog and take the last verification code
-    await newPage.goto("http://localhost:8025/");
+    await newPage.goto(`http://${process.env.MAIL_SERVER}:8025/`);
```

Comment:
This change breaks the login in e2e tests, because MAIL_SERVER variable cannot be 'localhost', it needs to be 0.0.0.0 in order to be able to send verification emails locally. 

After updating MAIL_SERVER to 0.0.0.0 the verification email is sent successfully but this url becomes 'http://0.0.0.0:8025' and the email verification fails in e2e tests.